shell/platform/tizen/flutter_tizen.cc

```diff
@@ -71,7 +71,12 @@ void FlutterDesktopEngineShutdown(FlutterDesktopEngineRef engine_ref) {
 
 void* FlutterDesktopPluginRegistrarGetNativeWindow(
     FlutterDesktopPluginRegistrarRef registrar) {
-  return registrar->engine->view()->window()->GetWindowHandle();
+  auto view_base = registrar->engine->view()->tizen_view();
```

Comment:
In this case, we prefer explicit types instead of `auto`.
see https://github.com/flutter-tizen/engine/pull/275

Reply:
I fixed it. Thank you